File: MasterErp.Angular/src/app/components/Finance/GeneralAccounts/models/GeneralAccounts/ReceiveReceipt.ts
export interface ReceiveReceipt {
    receiveReceiptId: number;
    receiptNumber: number;
    receiptLedgerId: number;
    cashReceiptTypeId: number;
    currencyId: number;
    receivedById: number | null;
    safeId: number | null;
    totalAmount: number;
    isLocked: boolean;
    isCancelled: boolean;
    loanNumber: string;
    ledgerNumber: string;
    justmentNumber: string;
    notes: string;
    insertUser: string;
    updateUser: string;
    releaseDate: string;
    insertDate: string;
    updateDate: string | null;
}

File: MasterErp.Entities/Models/ReceiveReceipt.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace MasterErp.Entities.Models
{
    public class ReceiveReceipt
    {

        [Key]
        public int ReceiveReceiptId { get; set; }
        public int ReceiptNumber { get; set; }
        public int ReceiptLedgerId { get; set; }
        public int ReceiveTypeId { get; set; }
        public int AccountId { get; set; }
        public int? ReceivedById { get; set; }
        public int? SafeId { get; set; }
        public int? AgencyTypeId { get; set; }
        public int? AgencyId { get; set; }
        public double MoneyAmount { get; set; }
        public bool IsLocked { get; set; }
        public bool IsCancelled { get; set; }
        public string Notes { get; set; }
        public string ChequeNumber { get; set; }
        public string DocNumber { get; set; }
        public string BenefitPerson { get; set; }

        public string InsertUser { get; set; }
        public string UpdateUser { get; set; }
        public DateTime ReleaseDate { get; set; }
        public DateTime InsertDate { get; set; }
        public DateTime? UpdateDate { get; set; }




    }
}
